mri/operators/base.py

```diff
@@ -0,0 +1,40 @@
+# #############################################################################
+#  pySAP - Copyright (C) CEA, 2017 - 2018                                     #
+#  Distributed under the terms of the CeCILL-B license,                       #
+#  as published by the CEA-CNRS-INRIA. Refer to the LICENSE file or to        #
+#  http://www.cecill.info/licences/Licence_CeCILL-B_V1-en.html for details.   #
+# #############################################################################
+
+
+class OperatorBase(object):
+    """ Base Operator class. Every operator inherits from this class,
+    to ensure that we have all the functions rightly implemented
+    as required by Modopt
+    """
+
+    def op(self, input):
```

Comment:
`input` is protected in Python. I think `x` or `data` are fine here.

Reply:
Oh damn I forgot, yes on it.